### PR TITLE
Add Code Blocks icon

### LINKED ISSUE #6637: Request: Code::Blocks
### Brand Name

Code::Blocks

### Website

https://www.codeblocks.org/

### Popularity Metric

The Alexa rank is 52,077. See https://www.alexa.com/siteinfo/codeblocks.org

### Official Resources for Icon and Color

Two variants of icons:
code1
code2

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by zrfisaac**
codeblocks

**Issue:** Close #6637

**Similarweb rank:** [90,808](https://www.codeblocks.org/)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Code::Blocks is a free C/C++ and Fortran IDE built to meet the most demanding needs of its users. It is designed to be very extensible and fully configurable.

Built around a plugin framework, Code::Blocks can be extended with plugins. Any kind of functionality can be added by installing/coding a plugin. For instance, event compiling and debugging functionality is provided by plugins!

### CODE REVIEW

**Comment by adamrusted** [comment]
I'd be happy with the treatment as is, looking at your comparison image @PeterShaggyNoble!

**Review by adamrusted — APPROVED** [review]
I'd be happy with the treatment as is, looking at your comparison image @PeterShaggyNoble!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2752,6 +2752,11 @@
             "hex": "222F29",
             "source": "https://www.codacy.com/blog/"
         },
+        {
+            "title": "Code Blocks",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:2756]:
```suggestion
            "title": "Code::Blocks",
```

**Review by PeterShaggyNoble — APPROVED** [review]
Just noticed that the name should be stylised as "Code::Blocks". Once that's updated, so, we can merge this in.

**Pull request merged**